mms_msg/sampling/pattern/meeting/state_based/__init__.py

```diff
@@ -4,3 +4,7 @@
 from . import sampler
 from . import transition_model
 from . import weighted_meeting_sampler
+
+import logging
+import sys
+logging.basicConfig(level=logging.INFO, stream=sys.stdout)
```

Comment:
Can you move this further up, into `mms_msg.__init__`? In the current location, the code will always be executed (it's imported through a chain of imports in the `__init__` files) but is much harder to find.

This is the official documentation for using loggers in a library, which explicitly states that a library should not configure any logging: https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library. But I agree that it's a good idea here to let INFO messages through when no logging is configured. Can you also write a comment why logging is configured here?